src/Symfony/Component/Config/Definition/ArrayNode.php

```diff
@@ -390,4 +390,12 @@ protected function mergeValues($leftSide, $rightSide)
 
         return $leftSide;
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function allowPlaceholders(): bool
+    {
+        return false;
```

Comment:
Why have you made it false by default for `ArrayNode`? It seems this will limit how much of a config you can change in other peoples bundles, i.e. you can only put `env(json:FOO)` if the owner things of you in advance.

Reply:
comments on closed PRs are likely to be lost, please consider opening a new issue/PR if you think something is buggy or can be improved. Thanks

Reply:
In this case i asked for it on another closed PR :P  But yeah.. not sure if there's any real issue now (hence i asked :D)

@mcfedr see https://github.com/symfony/symfony-docs/issues/8382#issuecomment-370135333 ,  tried to explain the general proces there.

Reply:
I see, makes sense, so for simple_array nodes it will accept `env(json:FOO)` but not when it should validate the structure.